crowdin.yml

```diff
@@ -0,0 +1,27 @@
+base_path: "."
+base_url: "https://neoforged.api.crowdin.com"
+
+# For interacting with Crowdin locally (via CLI), set these environment variables to their proper values
+project_id_env: "NEOFORGE_CROWDIN_PROJECT_ID"
+api_token_env: "NEOFORGE_CROWDIN_API_TOKEN"
+
+# Preserve directory structure (required by the use of the 'dest' field below)
+preserve_hierarchy: true
+
+# Prevent Crowdin-created commits from automatically triggering builds
+commit_message: "[ci skip]"
```

Comment:
This is at odds with your pr description

Reply:
In what way? I assume you're referring to this paragraph:
> To ease potential burden, commits made by the integration will have the `[no ci]` commit message, to prevent Actions from automatically running on them.

Reply:
Yeah, that. Ci skip != no ci

Reply:
I'm... not sure what you mean exactly. GitHub Actions recognizes [a variety of indicators to not run workflows](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs) (for `push` and `pull_request` events) on a commit, including `[no ci]`, `[ci skip]`, and `[skip ci]`.

Reply:
Ah, you're using as a generic rather than explicitly specifying what it adds?
It's just a nitpick on what the description says ;p

Reply:
Oh. I didn't realize I wrote the wrong thing between the file and the PR description. 😄 I'll correct the PR description to match.